p2p/libp2p/constants.go

```diff
@@ -0,0 +1,11 @@
+package libp2p
+
+import (
+	"github.com/ElrondNetwork/elrond-go-p2p/libp2p"
+)
+
+// ListenAddrWithIp4AndTcp defines the listening address with ip v.4 and TCP
```

Comment:
we can move all these outside the libp2p package to be directly in elrond-go/p2p because the p2p package will no hold only the adapter funcs/structs/consts/errors and so on